.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
+orbs:
+    node: circleci/node@4.4.0
+jobs:
+    lint:
+        executor: node/default
+        steps:
+            - checkout
+            - run: npm install --force
```

Comment:
is the `--force` required given that we're always doing a fresh install?

Reply:
Without the `--force`, this error appear :

`npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
npm ERR! Maximum call stack size exceeded
`

Reply:
Would work if you regenerate the lock file with a more recent version of npm?

Reply:
The problem is that I am running on the latest stable version of npm which generates lockfiles on version 2 and CircleCI is on npm version 6 which doesn't support lockfiles on version 2.
Should we update the npm version each time then ?

Reply:
let's chat about it. I think we should maybe collectively move to npm 7 across projects.